lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1989,6 +1990,20 @@ SwiftASTContext::CreateInstance(lldb::LanguageType language, Module &module,
   {
     LLDB_SCOPED_TIMERF("%s (getStdlibModule)", m_description.c_str());
     const bool can_create = true;
+
+    // Report progress on module importing by using a callback function in swift::ASTContext
+    Progress progress("Importing Swift standard library modules");
```

Comment:
Are there multiple standard library modules or is there one? I know it might be pulling in other modules as dependencies. I guess we could sidestep this by omitting "modules" at the end. @adrian-prantl WDYT?

Reply:
ping @chelcassanova

Reply:
"Modules" can be omitted here for clarity

Reply:
I don't understand ... you say it can be omitted but it's still in the PR 🧐 ...

```suggestion
    Progress progress("Importing Swift standard library");
```